### PR TITLE
Doc update to wildcard support match label selector

### PR DESCRIPTION
**Pull request by vyankyGH**
Signed-off-by: Vyankatesh   <vyankateshkd@gmail.com>
closes https://github.com/kyverno/website/issues/424

### CODE REVIEW

**Review by chipzoller — CHANGES_REQUESTED** [review]
Please move this under the ## Match Statements heading after the sentence "As of Kyverno 1.5.0, wildcards are supported in the kinds field allowing you to match on every resource type in the cluster."

File: content/en/docs/Writing policies/match-exclude.md [hunk 1]
```diff
@@ -160,6 +160,14 @@ Here are some other examples of `match` statements.
 This is an example that selects a Deployment **OR** a StatefulSet with a label `app=critical`.
 
 Condition checks inside the `resources` block follow the logic "**AND across types but an OR within list types**". For example, if a rule match contains a list of kinds and a list of namespaces, the rule will be evaluated if the request contains any one (OR) of the kinds AND any one (OR) of the namespaces. Conditions inside `clusterRoles`, `roles`, and `subjects` are always evaluated using a logical OR operation, as each request can only have a single instance of these values.
+Also match label selector support wildcard in key or value.
```
Comment (realshuting) [inline content/en/docs/Writing policies/match-exclude.md:163]: Can we be specific about the paths? We can list all impacted paths by `pol.spec.match.xxx.xxx`.

**Review by realshuting — CHANGES_REQUESTED** [review]

File: content/en/docs/Writing policies/match-exclude.md [hunk 2]
```diff
@@ -120,6 +120,14 @@ spec:
 ```
 
 As of Kyverno 1.5.0, wildcards are supported in the `kinds` field allowing you to match on every resource type in the cluster.
+Also, Wildcard supported at pol.spec.match.resources.selector.matchLabels in key as well as value of labels.
```
Comment (realshuting) [inline content/en/docs/Writing policies/match-exclude.md:123]: What about `pol.spec.match.any(all).xxx.xxx` and `pol.spec.exclude.xxx.xxx`?

Can you also wrap paths with ``?

Comment (vyankyGH) [reply content/en/docs/Writing policies/match-exclude.md:123]: got it. I miss any/all again. Will update.

File: content/en/docs/Writing policies/match-exclude.md [hunk 3]
```diff
@@ -120,6 +120,19 @@ spec:
 ```
 
 As of Kyverno 1.5.0, wildcards are supported in the `kinds` field allowing you to match on every resource type in the cluster.
+Also, Selector labels supported Wildcard `(* or ?)` for key as well as value on following paths.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:123]: "Selector labels support wildcards (`*` or `?`) for keys as well as values in the following paths."

File: content/en/docs/Writing policies/match-exclude.md [hunk 4]
```diff
@@ -120,6 +120,19 @@ spec:
...
+* `pol.spec.match.resources.selector.matchLabels`
```
Comment (chipzoller) [inline content/en/docs/Writing policies/match-exclude.md:124]: Remove the `pol.spec.` portion, please. Just start with either `match` or `exclude`.

**Review by chipzoller — CHANGES_REQUESTED** [review]
See comments.

File: content/en/docs/Writing policies/match-exclude.md (hunk 4 above)
Comment (vyankyGH) [reply content/en/docs/Writing policies/match-exclude.md:124]: yup

File: content/en/docs/Writing policies/match-exclude.md [hunk 5]
```diff
@@ -120,6 +120,19 @@ spec:
 ```
 
 As of Kyverno 1.5.0, wildcards are supported in the `kinds` field allowing you to match on every resource type in the cluster.
+Selector labels support wildcards `(* or ?)` for keys as well as values in the following paths.
+* `match.resources.selector.matchLabels`
+* `exclude.resources.selector.matchLabels`
```
Comment (realshuting) [inline content/en/docs/Writing policies/match-exclude.md:125]: Missing `exclude.any.resources.selector.matchLabels`, `exclude.all.resources.selector.matchLabels`?

Comment (vyankyGH) [reply content/en/docs/Writing policies/match-exclude.md:125]: added.

**Review by realshuting — APPROVED** [review]

**Comment by chipzoller** [comment]
Needs milestone assignment.

**Comment by realshuting** [comment]
Added to 1.5.3.

**Comment by vyankyGH** [comment]
> Added to 1.5.3, can we merge this?

yes

**Review by chipzoller — APPROVED** [review]

**Pull request merged**